src/vue/modules/dividend-form/index.vue

```diff
@@ -0,0 +1,530 @@
+    isSignersLoaded: false,
+  }),
+  computed: {
+    ...mapGetters('dividend-form', {
+      accountId: types.accountId,
+      balances: types.balances,
+    }),
+  },
+  watch: {
+    'form.ownedAsset.code' () {
+      this.tryGetSigners()
+    },
+  },
+  validations () {
+    return {
+      form: {
+        asset: { required },
+        amount: {
+          required,
+          amount,
+          noMoreThanAvailableOnBalance: noMoreThanAvailableOnBalance(
+            this.form.asset.balance.value
+          ),
+          maxDecimalDigitsCount: maxDecimalDigitsCount(
+            this.config.decimalPoints
+          ),
+        },
+      },
+    }
+  },
+  async created () {
+    initApi(this.wallet, this.config)
+
+    this.setAccountId(this.wallet.accountId)
+    await this.loadBalances()
+    this.isInitialized = true
+    try {
+      await this.initAssetSelector()
+      this.isLoaded = true
+    } catch (error) {
+      this.isFailed = true
+      ErrorHandler.processWithoutFeedback(error)
+    }
+  },
+  methods: {
+    ...mapMutations('dividend-form', {
+      setAccountId: types.SET_ACCOUNT_ID,
+    }),
+    ...mapActions('dividend-form', {
+      loadBalances: types.LOAD_BALANCES,
+    }),
+    async submit () {
+      this.disableForm()
+      try {
+        if (!this.isSignersLoaded) {
+          Bus.error('dividend-form.failed-load-signers')
+          return false
+        }
+        const operations = await this.getTransferOperations()
+        await api().postOperations(...operations)
+        await this.loadBalances()
+        await this.getSigners()
+        await this.reinitAssetSelector()
+        Bus.success('dividend-form.dividend-success')
+        this.$emit(EVENTS.operationSubmitted)
+      } catch (e) {
+        ErrorHandler.process(e)
+      }
+      this.enableForm()
+    },
+    async getSigners () {
+      this.isSignersLoaded = false
+      try {
+        const { data } = await api().getWithSignature(`/${HORIZON_VERSION_PREFIX}/balances`, {
+          filter: {
+            asset: this.form.ownedAsset.code,
+          },
+          include: ['state'],
+        })
+        this.balanceHolders = data
+          .map(item => new Balance(item))
+        await this.addAccountIdField()
+        this.balanceHolders = this.balanceHolders
+          .filter(balance => this.deleteOwnedBalance(balance))
+        this.isSignersLoaded = true
+      } catch (e) {
+        this.isSignersLoaded = false
+        ErrorHandler.processWithoutFeedback(e)
+      }
+      this.isSignersLoadPending = false
+    },
+    async addAccountIdField () {
+      await Promise.all(
+        this.balanceHolders
+          .map(async item => {
+            item.accountId = await this.getAccountId(item.id)
+            return item
+          })
+      )
+    },
+    deleteOwnedBalance (balance) {
+      if (balance.accountId === this.accountId) {
+        this.ownedUserBalance = this.getFullBalanceHolder(balance)
```

Comment:
What did you want to return here for your filter [here](https://github.com/tokend/web-client/pull/137/files#diff-23af4977e26eed3d7cb9ce46a739a981R306)?

Reply:
If owner balance exist, I calculate balance and write to a variable and return false else return true